### PR TITLE
:tada: Make Trivy Operator K8s vulnids consistent

### PR DESCRIPTION
**Pull request by manuel-sommer**
This PR fixes:
- trivy operator vulnids are malformed (e.g. KCV0092 --> AVD-KCV-0092)
- trivy operator vulnids can be resolved
- trivy operator secrets ruleIDs are not resolvable (are not listed in the aquasec vulnerability database), but were listed as a vulnid

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

File: dojo/tools/trivy_operator/uniform_vulnid.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import re
+
+
+class UniformTrivyVulnID:
+    def return_uniformed_vulnid(self, vulnid, test):
```
Comment (cneill) [inline dojo/tools/trivy_operator/uniform_vulnid.py:5]: The `test` argument appears unused?

```suggestion
    def return_uniformed_vulnid(self, vulnid):
```

File: dojo/tools/trivy_operator/compliance_handler.py [hunk 2]
```diff
@@ -54,6 +55,6 @@ def handle_compliance(self, benchmarkreport, test):
                         dynamic_finding=True,
                     )
                     if check_checkID:
-                        finding.unsaved_vulnerability_ids = [check_checkID]
+                        finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(check_checkID, test)]
```
Comment (cneill) [inline dojo/tools/trivy_operator/compliance_handler.py:58]:
```suggestion
                        finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(check_checkID)]
```

File: dojo/tools/trivy_operator/checks_handler.py [hunk 3]
```diff
@@ -47,6 +48,6 @@ def handle_checks(self, labels, checks, test):
                 tags=[resource_namespace],
             )
             if check_id:
-                finding.unsaved_vulnerability_ids = [check_id]
+                finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(check_id, test)]
```
Comment (cneill) [inline dojo/tools/trivy_operator/checks_handler.py:51]:
```suggestion
                finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(check_id)]
```

File: dojo/tools/trivy_operator/vulnerability_handler.py [hunk 4]
```diff
@@ -85,6 +86,6 @@ def handle_vulns(self, labels, vulnerabilities, test):
                 tags=finding_tags,
             )
             if vuln_id:
-                finding.unsaved_vulnerability_ids = [vuln_id]
+                finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(vuln_id, test)]
```
Comment (cneill) [inline dojo/tools/trivy_operator/vulnerability_handler.py:89]:
```suggestion
                finding.unsaved_vulnerability_ids = [UniformTrivyVulnID().return_uniformed_vulnid(vuln_id)]
```

**Comment by manuel-sommer** [comment]
@cneill , could we merge this in the upcomming release on Monday?

**Comment by manuel-sommer** [comment]
Friendly reminder @cneill

**Comment by manuel-sommer** [comment]
@Maffooch, could we introduce this PR to the upcomming release today?

**Comment by Maffooch** [comment]
Will need sign off from @cneill first, but the release will actually go out tomorrow, as today is a US holiday

**Review by cneill — APPROVED** [review]

**Pull request merged**